test/features/page_objects/poa.rb

```diff
@@ -0,0 +1,142 @@
+require_relative '../utils/index.rb'
+
+class POA
+
+  def initialize(driver)
+    @driver = driver
+  end
+
+  def file_upload
+    element = @driver.find_element(:css, '.onfido-sdk-ui-Uploader-uploadArea input[type="file"]')
+    @driver.execute_script("return arguments[0].setAttribute('style','display: true');", element)
+    @driver.execute_script("return arguments[0].value = '';", element) unless element.attribute('value').empty?
+    element
+  end
+
+  def uploaded_image
+    @driver.find_element(:css, '.onfido-sdk-ui-Confirm-CaptureViewer-image')
+  end
+
+  def verify_uk_address_title
+    @driver.find_element(:xpath, "//span[contains(.,'Let’s verify your UK address')]")
+  end
+
+  def need_a_doc_text
+    @driver.find_element(:xpath, "//p[contains(.,'You’ll need a document that:')]")
+  end
+
+  def poa_requirements
+    @driver.find_element(:xpath, "//span[contains(.,'Shows your current address')]")
+    @driver.find_element(:xpath, "//span[contains(.,'Matches the address you used on signup')]")
+    @driver.find_element(:xpath, "//span[contains(.,'Is your most recent document')]")
+  end
+
+  def start_verification_button
+    @driver.find_element(:xpath, "//button[contains(.,'Start verification')]")
+  end
+
+  ### Document selection screen elements
+  def document_selection_screen_title
+    @driver.find_element(:xpath, "//span[@class='onfido-sdk-ui-Title-titleSpan'][contains(.,'Select a UK document')]")
+  end
+
+  def document_selection_screen_subtitle
+    @driver.find_element(:xpath, "(//div[contains(.,'These are the documents most likely to show your current home address')])[9]")
+  end
+
+  def bank_building_statement_cell
```

Comment:
This can be refactored in the same way you did for `document_upload_intro_screen(title)`. You could call it something like `poa_document_cell(poa_doc)` and you will be able to delete the following methods
- `utility_bill_cell`
- `council_tax_letter_cell`
- `benefits_letter_cell`